src/map/clif.c

```diff
@@ -654,6 +655,21 @@ bool clif_send(const void* buf, int len, struct block_list* bl, enum send_target
 			}
 			break;
 
+		case CLAN:
+#if PACKETVER >= 20131223
```

Comment:
this date more than packets add time. this mean should works. but probably can works date bit smaller. Packets was added somewhere in 2012. But if it tested at least in 20131223  should be ok too. After pr will be fixed, some one can try guess smaller version